tests/test_curves.py

```diff
@@ -461,6 +493,24 @@ def test_asserts(self):
 class TestPlanarCurve:
     """Tests for FourierPlanarCurve class."""
 
+    @pytest.mark.unit
+    def test_center(self):
+        """Test center of curve."""
+        c = FourierPlanarCurve(center=[5, 4, 3], r_n=[1, 2, 0.5], basis="xyz")
```

Comment:
also add one with basis_in as rpz to increase test coverage

Reply:
That is already done in this test [a few lines below](https://github.com/PlasmaControl/DESC/blob/dd/curvature/tests/test_curves.py#L509). 

You may be confused because the kwarg is called `basis` in the `FourierPlanarCurve` class, but we call it `basis_in` in the compute functions to differentiate it from the other "basis" kwarg (the latter is really "basis_out").